docs/book/src/topics/windows.md

```diff
@@ -6,7 +6,7 @@ CAPZ enables you to create Windows Kubernetes clusters on Microsoft Azure. We re
 
 ### Using Containerd for Windows Clusters
 
-To deploy a cluster using Windows, use the [Windows Containerd flavor template](../../../../templates/cluster-template-machinepool-windows-containerd.yaml).
+To deploy a cluster using Windows, use the [Windows Containerd flavor template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/cluster-template-windows-containerd.yaml).
```

Comment:
Thanks for this correction! Please continue to use the relative link (`../` etc)

Reply:
relative links don't work from the book (https://capz.sigs.k8s.io/topics/windows.html), that's what's causing it to be broken

Reply:
for other pages we use the raw github link, eg. https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/c5a4d71190d633f16ffb22d933db8bfd4db0d267/docs/book/src/topics/cloud-provider-config.md?plain=1#L72